### ISSUE TITLE
Basic poses create too much distortion

### ISSUE THREAD

**Issue by aaronfjerstad** [animate1978/MB-Lab#241]
**Describe the bug**

**To Reproduce**
1. Create a basic character (any type) with no modifications. Include basic muscles and IK rig.
2. Use the hip controller to drop the character so the butt is on the floor, and adjust the feet slightly forward to reduce mesh penetration.
3. You'll already notice that the knees are far too rounded as in screenshot 1.
4. Shift the knees to the side slightly.
5. Notice the body horror that occurs in screenshot 2.

**Expected behavior**
One would expect basic posing of joints and limbs to work

**Screenshots**

![image](https://user-images.githubusercontent.com/4307419/75059388-17d8c380-549a-11ea-9569-f014f593db1f.png)
![image](https://user-images.githubusercontent.com/4307419/75059669-b402ca80-549a-11ea-8ca6-46f52a4a1841.png)

**Additional context**
I'm running version 1.7.7 on Blender 2.82

**Logs**
`mb_lab updater verbose is enabled
cls BA: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_install_popup'>
cls: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_install_popup'>
cls BA: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_check_now'>
cls: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_check_now'>
cls BA: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_update_now'>
cls: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_update_now'>
cls BA: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_update_target'>
cls: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_update_target'>
cls BA: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_install_manually'>
cls: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_install_manually'>
cls BA: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_updated_successful'>
cls: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_updated_successful'>
cls BA: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_restore_backup'>
cls: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_restore_backup'>
cls BA: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_ignore'>
cls: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_ignore'>
cls BA: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_end_background'>
cls: <class 'MB-Lab-1_7_7.addon_updater_ops.addon_updater_end_background'>
mb_lab Updater: Read in json settings from file`

**Comment by aaronfjerstad** [comment]
Note that I can get something closer if I don't use IK:

![image](https://user-images.githubusercontent.com/4307419/75061332-bfa3c080-549d-11ea-873d-e6a2889e2653.png)

**Comment by blaisefaint** [comment]
I'm curious if anyone has tried using Blender's new pose brush created by Pablo Dobarro.

Blender is also working on Animation 2020.

On the most recent Blender Today Live, Pablo Vazquez mentioned that someone is working on new tools to improve working with bvh files: https://www.youtube.com/watch?v=nMuCLduOmtE

The bottom line is that Manuel Bastioni designed these pose tools before these new tools came along, meaning the whole idea of posing in MB-Lab may be due for a complete overhaul.

**Comment by TetoTheSquirrelFox** [comment]
Yep. And I can add this : a addon to add rigify on a MB-Lab model already exists, and should be better that the one Manuel did for MB-Lab.
And you can use Blenrig too, it seems to be a very powerful tool.
But it's obvious that tools made by Manuel for rigging are a little outdated and must be upgraded. No offense, it's just that rigging tools are moving fast since 2~3 years.

**Comment by Noizirom** [comment]
We are looking at different options for the skeletal system. We are kind of
cautious at the moment because we're not sure what blender will do with
their animation system overhaul. It will affect how bones are animated. Jim
Kroovy has an animation system that he was willing to donate to the
project. You should check it out at
https://github.com/Jim-Kroovy/Mr-Mannequins-Tools

**Comment by animate1978** [comment]
In my opinion the IK in MB-Lab is less than desirable really, however my skills in that area are probably even less so. That being said we would need someone who is skilled.

I checked out the repo above. I remember seeing this before...
